build.sbt

```diff
@@ -78,9 +78,10 @@ lazy val semanticdb3 = crossProject
   .in(file("semanticdb/semanticdb3"))
   .settings(
     publishableSettings,
+    ignoreMimaSettings,
```

Comment:
@thesamet Are there best practices for achieving source/binary compatibility of ScalaPB-generated code during compatible Protobuf schema evolution?

For example, this pull request removes the `range` field and adds the `location` field to the `SymbolInformation` message. As far as I understand, this change is both backward and forward compatible in the sense that old Protobuf payloads are valid under the new schema and new Protobuf payloads are valid under the old schema.

However, the changes to the `SymbolInformation` Scala class generated by ScalaPB are not compatible (so I had to disable [MiMa](https://github.com/lightbend/migration-manager) for this project), which is regrettable. Can you provide recommendations for dealing with this?

Reply:
AFAIK there is no way around the binary compatibility issue as long as ScalaPB generates case classes. It has been suggested that we switch to normal classes, but there wasn't much of a demand for that.

Reply:
Thank you! I think we'll just bite the bullet for now, since this is an internal API anyway.